app/src/main/java/com/example/mvp/ui/user_activity/UserActivity.java

```diff
@@ -0,0 +1,102 @@
+package com.example.mvp.ui.user_activity;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+import android.widget.Toast;
+import com.example.mvp.data.model.User;
+import com.example.mvp.utils.Constants;
+import com.example.vando.mvp.R;
+import com.squareup.picasso.Picasso;
+
+public class UserActivity extends AppCompatActivity implements UserViewContract.View {
+    private TextView mTxtId, mTxtFistName, mTxtLastName, mTxtEmail;
+    private ImageView mImgView;
+    private UserViewPresenter mUserPresenter;
+    private ProgressBar mProgressbar;
+    private LinearLayout mLayoutMaster;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_user_view);
+        mUserPresenter = UserViewPresenter.getInstance();
+        mUserPresenter.setmUserViewCallBack(this);
+        Intent intent = getIntent();
+        int id = intent.getIntExtra(Constants.STRING_KEY_INTENT, -1);
+        initView();
+        if (id >= 0)
+            mUserPresenter.getUserById(id);
+    }
+
+    @Override
+    public void updateView(User user) {
+        Picasso.get().load(user.getmAvatar())
+                .centerCrop()
+                .fit()
+                .into(mImgView);
+        mTxtId.setText(user.getmId());
+        mTxtFistName.setText(user.getmFirstName());
+        mTxtLastName.setText(user.getmLastName());
+        mTxtEmail.setText(user.getmEmail());
+    }
+
+
+    @Override
+    public void unShowProgress() {
+        mProgressbar.setVisibility(View.GONE);
+        mLayoutMaster.setVisibility(View.VISIBLE);
+    }
+
+    @Override
+    public void showProgress() {
+        mProgressbar.setVisibility(View.VISIBLE);
+        mLayoutMaster.setVisibility(View.INVISIBLE);
+    }
+
+    @Override
+    public void updateUser(User user) {
+        if (user != null) {
+            mTxtEmail.setText(user.getmEmail());
+            mTxtLastName.setText(user.getmLastName());
+            mTxtFistName.setText(user.getmFirstName());
+            mTxtId.setText(user.getmId() + "");
+            Picasso.get().load(user.getmAvatar() + "")
+                    .centerCrop()
+                    .fit()
+                    .into(mImgView);
+        } else {
+            Toast.makeText(this, "Không có User này!!!", Toast.LENGTH_SHORT).show();
```

Comment:
hard code, có thể lưu trong file strings.xml hoặc tạo biến final
```suggestion
            Toast.makeText(this, getString(R.string.notification_...), Toast.LENGTH_SHORT).show();
```